@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -113,7 +116,7 @@ export class InterpolatedClipPlanesEvaluator implements ClipPlanesEvaluator {
             // the camera is looking at the center of the scene.
             const r = EarthConstants.EQUATORIAL_RADIUS;
             const d = camera.position.length();
-            const alpha = Math.asin(r / d);
+            const alpha = Math.atan(r / d);
```

Comment:
This alpha is the angle from the camera position to the tangent point on the sphere right?

   `          
   
                    , - ~ ~ ~ - ,
              , '                     '   ,        
            ,           .                  ,      
           ,            .                    ,   
          ,             .                     ,  
          ,             . O '                , 
          ,               | .                 ,  
           ,              |   .  r            ,
            ,             |      .           ,
              ,           |        ,       '_____ far
          /       ' -_, _ | _ ,  ' @ / T
              near        |        /
                          |       / t
                      d   |     /
                          |   /
                          |  / <--- alpha
                           E
`
The angle between the earth radius (r) and the tangent ray (t) is a right angle (@), therefore they are the legs of a right triangle. Alpha is the angle between the hypothenuse (d) and the tangent ray , so it's sinus is the opposite leg divided by the hypothenuse:  alpha = asin(r/d)

Reply:
Thank you, yes I interpreted the ascii diagram wrong and came to the wrong conclusion, I will revert.